### ISSUE TITLE
the sample"authentication-profile"does not work it

### ISSUE THREAD

**Issue by HeJiao12** [Esri/arcgis-runtime-samples-android#305]
I cloned the sample"authentication-profile" to my project ,but I can not login in with my arcgis online account,besides, i am sure no problem with the username and password .I am looking forward to your response onine.

**Comment by doneill** [comment]
@HeJiao12 Try updated the portal url to use `https` on line 58:  

```java
        // Set loginRequired to true always prompt for credential,
        // When set to false to only login if required by the portal
        final Portal portal = new Portal("https://www.arcgis.com", true);
```

I have a fix for this with PR #306

**Comment by doneill** [comment]
Closing as verified fixed
